pkg/mutator/handler.go

```diff
@@ -97,7 +99,7 @@ func extractName(request *v1beta1.AdmissionRequest) string {
 	return "<unknown>"
 }
 
-func writeResponse(mutator Mutator, writer http.ResponseWriter, response *v1beta1.AdmissionResponse) {
+func writeResponse(mutator generic.Logger, writer http.ResponseWriter, response *v1beta1.AdmissionResponse) {
```

Comment:
Variable name `mutator` _looks_ weird for a logger :stuck_out_tongue:

Reply:
true that :smile:

Reply:
leftovers I think